### ISSUE TITLE
GeneratedIntColumn, GeneratedIntColumn, GeneratedDateTimeColumn etc not found in generated file

### ISSUE THREAD

**Issue by krunduev** [simolus3/moor#1449]
On new environment I have 125 errors in generated file like this: 

```
Undefined class 'GeneratedTextColumn'.
Undefined class 'GeneratedIntColumn'.
Undefined class 'GeneratedDateTimeColumn'.
```

Before all worked fine. Just new environment, nothing changed in code.

  moor 4.4.1 (4.5.0 available)
  moor_generator 4.4.1 (4.5.1 available)

(Can't update to latest due conflicts with flutter_test and flutter_localizations (which depends on meta 1.3.0))

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.3.1 20E241 darwin-arm, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] IntelliJ IDEA Ultimate Edition (version 2021.2)
[✓] VS Code (version 1.60.0)
[✓] Connected device (3 available)

• No issues found!

**Comment by krunduev** [comment]
I have added example.dart to project. And after 
`flutter packages pub run build_runner watch`
such unknown fields disappeared.

Very strange behaviour.

**Comment by krunduev** [comment]
Now I deleted example.dart and all generated file. Again all ok.
Yesterday I spend more than 2 hours trying ti solve problem. Regenerating, change versions etc.
It was stable. But today it just disappeared.

Yesterday errors:


  ```
  lib/data/db/moor_database.g.dart:326:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _book_id;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:328:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get book_id => _book_id ??= _constructBookId();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:329:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructBookId() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:335:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _for_year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:337:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get for_year => _for_year ??= _constructForYear();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:338:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructForYear() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:347:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _for_month;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:349:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get for_month => _for_month ??= _constructForMonth();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:350:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructForMonth() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:359:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _for_day;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:361:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get for_day => _for_day ??= _constructForDay();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:362:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructForDay() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:371:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _lang;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:373:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get lang => _lang ??= _constructLang();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:374:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructLang() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:380:3: Error: Type 'GeneratedBoolColumn' not found.
      GeneratedBoolColumn _is_israel;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:382:3: Error: Type 'GeneratedBoolColumn' not found.
      GeneratedBoolColumn get is_israel => _is_israel ??= _constructIsIsrael();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:383:3: Error: Type 'GeneratedBoolColumn' not found.
      GeneratedBoolColumn _constructIsIsrael() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:392:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _html;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:394:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get html => _html ??= _constructHtml();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:395:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructHtml() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:405:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _updated_date;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:407:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn get updated_date =>
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:409:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _constructUpdatedDate() {
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:695:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _id;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:697:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get id => _id ??= _constructId();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:698:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructId() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:707:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _lang;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:709:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get lang => _lang ??= _constructLang();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:710:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructLang() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:716:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _html;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:718:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get html => _html ??= _constructHtml();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:719:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructHtml() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:729:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _updated_date;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:731:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn get updated_date =>
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:733:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _constructUpdatedDate() {
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1114:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _book_id;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1116:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get book_id => _book_id ??= _constructBookId();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1117:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructBookId() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1123:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _for_year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1125:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get for_year => _for_year ??= _constructForYear();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1126:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructForYear() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1135:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _for_month;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1137:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get for_month => _for_month ??= _constructForMonth();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1138:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructForMonth() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1147:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _for_day;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1149:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get for_day => _for_day ??= _constructForDay();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1150:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructForDay() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1159:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _lang;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1161:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get lang => _lang ??= _constructLang();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1162:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructLang() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1168:3: Error: Type 'GeneratedBoolColumn' not found.
      GeneratedBoolColumn _is_israel;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1170:3: Error: Type 'GeneratedBoolColumn' not found.
      GeneratedBoolColumn get is_israel => _is_israel ??= _constructIsIsrael();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1171:3: Error: Type 'GeneratedBoolColumn' not found.
      GeneratedBoolColumn _constructIsIsrael() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1180:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _tehelim;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1182:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get tehelim => _tehelim ??= _constructTehelim();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1183:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructTehelim() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1192:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _added_date;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1194:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn get added_date =>
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1196:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _constructAddedDate() {
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1448:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _date;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1450:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get date => _date ??= _constructDate();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1451:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructDate() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1460:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1462:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get year => _year ??= _constructYear();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1463:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructYear() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1472:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _name;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1474:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn get name => _name ??= _constructName();
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1475:3: Error: Type 'GeneratedTextColumn' not found.
      GeneratedTextColumn _constructName() {
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1661:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1663:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn get year => _year ??= _constructYear();
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1664:3: Error: Type 'GeneratedIntColumn' not found.
      GeneratedIntColumn _constructYear() {
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1673:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _updateTime;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1675:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn get updateTime =>
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1677:3: Error: Type 'GeneratedDateTimeColumn' not found.
      GeneratedDateTimeColumn _constructUpdateTime() {
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:326:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _book_id;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:330:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn('book_id', $tableName, false,
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:335:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _for_year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:339:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:347:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _for_month;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:351:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:359:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _for_day;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:363:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:371:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _lang;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:375:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn('lang', $tableName, false,
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:380:3: Error: 'GeneratedBoolColumn' isn't a type.
      GeneratedBoolColumn _is_israel;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:384:12: Error: The method 'GeneratedBoolColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedBoolColumn'.
        return GeneratedBoolColumn(
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:392:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _html;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:396:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn(
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:405:3: Error: 'GeneratedDateTimeColumn' isn't a type.
      GeneratedDateTimeColumn _updated_date;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:410:12: Error: The method 'GeneratedDateTimeColumn' isn't defined for the class '$HitasTextsTable'.
     - '$HitasTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedDateTimeColumn'.
        return GeneratedDateTimeColumn(
               ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:695:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _id;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:699:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$TehelimTextsTable'.
     - '$TehelimTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:707:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _lang;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:711:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$TehelimTextsTable'.
     - '$TehelimTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn('lang', $tableName, false,
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:716:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _html;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:720:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$TehelimTextsTable'.
     - '$TehelimTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn(
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:729:3: Error: 'GeneratedDateTimeColumn' isn't a type.
      GeneratedDateTimeColumn _updated_date;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:734:12: Error: The method 'GeneratedDateTimeColumn' isn't defined for the class '$TehelimTextsTable'.
     - '$TehelimTextsTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedDateTimeColumn'.
        return GeneratedDateTimeColumn(
               ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1114:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _book_id;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1118:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn('book_id', $tableName, false,
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1123:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _for_year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1127:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1135:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _for_month;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1139:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1147:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _for_day;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1151:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1159:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _lang;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1163:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn('lang', $tableName, false,
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1168:3: Error: 'GeneratedBoolColumn' isn't a type.
      GeneratedBoolColumn _is_israel;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1172:12: Error: The method 'GeneratedBoolColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedBoolColumn'.
        return GeneratedBoolColumn(
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1180:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _tehelim;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1184:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1192:3: Error: 'GeneratedDateTimeColumn' isn't a type.
      GeneratedDateTimeColumn _added_date;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1197:12: Error: The method 'GeneratedDateTimeColumn' isn't defined for the class '$BookmarksTable'.
     - '$BookmarksTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedDateTimeColumn'.
        return GeneratedDateTimeColumn(
               ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1448:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _date;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1452:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$ParshaInfosTable'.
     - '$ParshaInfosTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1460:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1464:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$ParshaInfosTable'.
     - '$ParshaInfosTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1472:3: Error: 'GeneratedTextColumn' isn't a type.
      GeneratedTextColumn _name;
      ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1476:12: Error: The method 'GeneratedTextColumn' isn't defined for the class '$ParshaInfosTable'.
     - '$ParshaInfosTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedTextColumn'.
        return GeneratedTextColumn('name', $tableName, false,
               ^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1661:3: Error: 'GeneratedIntColumn' isn't a type.
      GeneratedIntColumn _year;
      ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1665:12: Error: The method 'GeneratedIntColumn' isn't defined for the class '$ParshaInfoUpdatesTable'.
     - '$ParshaInfoUpdatesTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedIntColumn'.
        return GeneratedIntColumn(
               ^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1673:3: Error: 'GeneratedDateTimeColumn' isn't a type.
      GeneratedDateTimeColumn _updateTime;
      ^^^^^^^^^^^^^^^^^^^^^^^
    lib/data/db/moor_database.g.dart:1678:12: Error: The method 'GeneratedDateTimeColumn' isn't defined for the class '$ParshaInfoUpdatesTable'.
     - '$ParshaInfoUpdatesTable' is from 'package:hitas/data/db/moor_database.dart' ('lib/data/db/moor_database.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'GeneratedDateTimeColumn'.
        return GeneratedDateTimeColumn(
               ^^^^^^^^^^^^^^^^^^^^^^^

   
```

**Comment by krunduev** [comment]
Yesterday it generates as 
`    GeneratedTextColumn _book_id;
`
Today 
`  GeneratedColumn<String> _book_id;
`

I DO NOT CHANGE ANYTHING IN CODE. Even did not restart computer and Android studio.

**Comment by simolus3** [comment]
Sorry about this. In `moor` and `moor_generator` version `4.5.0`, I changed how the column implementation works. I did not consider this to be a breaking change as one wouldn't notice when re-running the build with the latest generator. In hindsight, that was not a good call.

I think the best way to fix this is to change your dependencies to `moor: ^4.5.0` and `moor_generator: ^4.5.0` so that all environments will have a consistent generator and runtime  version.
